mbtiles/src/errors.rs

```diff
@@ -57,7 +57,7 @@ pub enum MbtError {
     #[error("The destination file {0} is not empty. Some operations like creating a diff file require the destination file to be non-existent or empty.")]
     NonEmptyTargetFile(PathBuf),
 
-    #[error("The file {0} does not have the required uniqueness constraint")]
+    #[error("The file {0} does not have the required uniqueness constraint. Try adding a unique index on the `map` or `tiles` table.")]
```

Comment:
Perhaps link to https://maplibre.org/martin/mbtiles-schema.html - and also update that page to include some information about performance?  Note that `tiles` is the only official requirement of an mbtiles, but it could be either a table or a view - and it would be difficult to explain all this in a few lines.